### DISCUSSION TITLE
how to remove red from a hint book?

### DISCUSSION THREAD

**Discussion by dmd** [ImageMagick/ImageMagick#6965] [Help] [answered]
### ImageMagick version

7.1.1-21

### Operating system, version and so on

MacOS

### Description

Game hint books from the 80s and 90s often used red text overlay, which would be "removed" by viewing the page through a red plastic filter. 

[Here's one example](https://www.mocagh.org/lucasfilm/mi2-hintbook.pdf), and a sample:

<img width="482" alt="image" src="https://github.com/ImageMagick/ImageMagick/assets/41439/3f539462-23d6-4faa-aa29-6c50076225fc">

How could I use imagemagick to remove the red, making the underlying blue text legible? (The above linked PDF in particular is the one I'm most interested in.)

**Comment by snibgo** [comment, marked as answer]
I suggest extracting the images with "pdfimages", then:
```
magick abc-010.jpg -channel R -separate +channel x.png
```
The result, x.png, is legible.

**Comment by dmd** [comment]
Perfect, thank you!
